TechDoc-Assistance-Program.md

```diff
@@ -0,0 +1,55 @@
+    1. Contributor documentation: Documentation about the project, aimed at project contributors and describing procedures, infrastructure, and customs for doing project work. This includes artifacts that define procedures and governance; recruit, orient, and train project contributors; and name responsible parties (project leaders, often generically called *maintainers*).
+    1. Website: The technical infrastructure behind the documentation and the project's web presence, including website generation, versioning, SEO, analytics, and security.
+1. Collaborates with project leadership to identify user roles and objectives for software users. 
+1. Proposes changes, if necessary, to the organization and content of the documentation to close gaps with the proposed maturity level.
+1. Writes an implementation plan describing improvements that address the gaps identified in the analysis.
+
+## Phase 2: Backlog Creation
+
+Once a high-level improvement plan has been written and approved, the tech writer analyzes the proposed changes to create a backlog of actionable, individual writing assignments and other tasks to improve the documentation. These tasks should have two primary characteristics:
+- As much as possible, they should be independent of each other so they can be completed in any order by any combination of contributors; and
+- They should be time-constrained. If possible, large tasks should be broken down into smaller, independent tasks that require hours, not days, to complete.
+
+## Phase 3: Documentation Improvement
+
+Community members work on the issues created in the previous phase. Ideally, tech writers are available to advise contributors and edit work, especially if the contributing community members are not trained technical writers. Remember that the training courses in Phase 0 are available to prepare contributors with general knowledge of the process.
+
+We know that recruiting contributors to write documentation can be difficult. While this is largely the responsibility of the project leadership, CNCF is actively working on ways to encourage doc contributions. For example, creating a backlog of time-bounded issues is an attempt to lower barriers, both psychological and logistical, to documentation creation and maintenance. 
+
+## Phase 4: Impact Analysis
+
+Measurement is vital to any process improvement effort, and this is currently the weakest point of the TechDoc Assistance Program. We do no offer any tools to measure the impact of doc improvements, although we can assure you that they are profound. Projects are encouraged to collect metrics on documentation effectiveness. 
+
+As a basic gauge the effectiveness of the documentation effort, projects can track the issue backlog. This can provide a guide to progress on leveling the documentation maturity to that of the project.
```

Comment:
Hmm, I'm not sure about this paragraph. How about (or just drop the paragraph IMHO):

```suggestion
Projects can also track the issue backlog as a means of measuring the increased documentation maturity.
```

Reply:
Agreed.